src/roles/vengefulghost.py

```diff
@@ -105,28 +105,15 @@ def on_del_player(evt, var, user, nickrole, nicktpls, death_triggers):
         user.send(messages["vengeful_turn"].format(GHOSTS[user]))
         debuglog(user.nick, "(vengeful ghost) TRIGGER", GHOSTS[user])
 
-@event_listener("rename_player")
-def on_rename(evt, cli, var, prefix, nick):
-    kvp = []
-    for a,b in KILLS.items():
-        if a == prefix:
-            a = nick
-        if b == prefix:
-            b = nick
-        kvp.append((a,b))
-    KILLS.update(kvp)
-    if prefix in KILLS:
-        del KILLS[prefix]
-
 @event_listener("transition_day_begin", priority=6)
-def on_transition_day_begin(evt, cli, var):
+def on_transition_day_begin(evt, var):
     # select a random target for VG if they didn't kill
-    wolves = set(list_players(var.WOLFTEAM_ROLES))
-    villagers = set(list_players()) - wolves
+    wolves = set(get_players(var.WOLFTEAM_ROLES))
+    villagers = set(get_players()) - wolves
     for ghost, target in GHOSTS.items():
         if target[0] == "!" or ghost.nick in var.SILENCED:
             continue
-        if ghost.nick not in KILLS:
+        if ghost not in KILLS:
             choice = set()
             if target == "wolves":
                 choice = wolves.copy()
```

Comment:
Why are we doing .copy() here? (and 2 lines below). I know this wasn't your thing, but the copy seems superfluous.